File: src/Tappxi/Bundle/ApiBundle/Entity/Stand.php
<?php

namespace Tappxi\Bundle\ApiBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Tappxi\Bundle\ApiBundle\Entity\Stand
 *
 * @ORM\Table(name="stand")
 * @ORM\Entity
 */
class Stand
{
    /**
     * @var integer $id
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string $name
     *
     * @ORM\Column(name="name", type="string", length=60, nullable=false)
     */
    private $name;

    /**
     * @var float $startFare
     *
     * @ORM\Column(name="start_fare", type="float", nullable=false)
     */
    private $startFare;

    /**
     * @var integer $status
     *
     * @ORM\Column(name="status", type="smallint", nullable=false)
     */
    private $status;

    /**
     * @var Address
     *
     * @ORM\ManyToOne(targetEntity="Address")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="address_id", referencedColumnName="id")
     * })
     */
    private $address;



    /**
     * Get id
     *
     * @return integer 
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set name
     *
     * @param string $name
     * @return Stand
     */
    public function setName($name)
    {
        $this->name = $name;
    
        return $this;
    }

    /**
     * Get name
     *
     * @return string 
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * Set startFare
     *
     * @param float $startFare
     * @return Stand
     */
    public function setStartFare($startFare)
    {
        $this->startFare = $startFare;
    
        return $this;
    }

    /**
     * Get startFare
     *
     * @return float 
     */
    public function getStartFare()
    {
        return $this->startFare;
    }

    /**
     * Set status
     *
     * @param integer $status
     * @return Stand
     */
    public function setStatus($status)
    {
        $this->status = $status;
    
        return $this;
    }

    /**
     * Get status
     *
     * @return integer 
     */
    public function getStatus()
    {
        return $this->status;
    }

    /**
     * Set address
     *
     * @param Tappxi\Bundle\ApiBundle\Entity\Address $address
     * @return Stand
     */
    public function setAddress(\Tappxi\Bundle\ApiBundle\Entity\Address $address = null)
    {
        $this->address = $address;
    
        return $this;
    }

    /**
     * Get address
     *
     * @return Tappxi\Bundle\ApiBundle\Entity\Address 
     */
    public function getAddress()
    {
        return $this->address;
    }
}

File: android-tappxi/src/cmdf2/tappxi/model/bean/Stand.java
package cmdf2.tappxi.model.bean;

import java.io.Serializable;

import org.json.JSONException;
import org.json.JSONObject;

public class Stand implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private int id;
	private String name;
	private float startFare;
	private int status;
	private Address address;
	private String phone;
	
	public Stand(int id, String name, float startFare, int status,
			Address address, String phone) {
		super();
		this.id = id;
		this.name = name;
		this.startFare = startFare;
		this.status = status;
		this.address = address;
		this.phone = phone;
	}
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public float getStartFare() {
		return startFare;
	}
	public void setStartFare(float startFare) {
		this.startFare = startFare;
	}
	public int getStatus() {
		return status;
	}
	public void setStatus(int status) {
		this.status = status;
	}
	public Address getAddress() {
		return address;
	}
	public void setAddress(Address address) {
		this.address = address;
	}
	
	public String getPhone() {
		return phone;
	}

	public void setPhone(String phone) {
		this.phone = phone;
	}

	public String toString() {
		return getName();
	}

	public static Stand fromJSONObject(JSONObject object) throws JSONException {
		return new Stand(object.getInt("id"), object.getString("name"), (float)object.getDouble("start_fare"), object.getInt("status"), Address.fromJSONObject(object.getJSONObject("address")), object.getString("phone"));
	}
}
